docs/src/tutorials/raise-alarm.md

```diff
@@ -68,6 +68,14 @@ Thin-edge.io doesn't keep any history of all alarms raised on an alarm topic.
 
 An already raised alarm can be cleared by sending an empty message with retained flag enabled to the same alarm topic on which the original alarm was raised.
 
+Topic:
+`tedge/alarms/<severity-level>/<alarm-type>`
+
+Payload:
+```
+""
+```
```

Comment:
Technically the payload should be empty, not a literal `""` which this line suggests.

```suggestion
Payload:

*Empty payload*
```

Reply:
Added an example, I think this is more clearer now.